src/main/java/se/liu/ida/hefquin/engine/federation/access/Neo4JException.java

```diff
@@ -0,0 +1,7 @@
+package se.liu.ida.hefquin.engine.federation.access;
+
+public class Neo4JException extends RuntimeException {
```

Comment:
On the contrary, if I just extend the `Exception` class I am forced to either declare or handle the exception when thrown. Maybe I'm not understanding what you mean.

Reply:
Yes, that's exactly my point. I want you/us to be forced to declare and to handle our exceptions. So, what I wanted to say is that I would prefer you to just extend the `Exception` class rather than `RuntimeException`.